frontend/src/views/Signin.vue

```diff
@@ -32,6 +32,10 @@
     ></v-text-field>
     <v-btn @click="signin">Sign In</v-btn>
   </v-form>
+  <br />
```

Comment:
We hate line breaks.
Maybe a v-card could do the trick.

Reply:
I'm not the frontend guy